client/src/document/organisms/sidebar/index.scss

```
@@ -254,6 +254,7 @@
   }
   @media (min-width: $screen-md) {
     display: flex;
+    position: sticky;
```

No need to make it sticky, as we just want to avoid `.sidebar` from extending beyond its parent (`.sidebar-container`):
```suggestion
    position: relative;
```